src/kubernetes/api_watcher.rs

```diff
@@ -64,11 +64,20 @@ where
         emit!(internal_events::RequestPrepared { request: &request });
 
         // Send request, get response.
-        let response = self
-            .client
-            .send(request)
-            .await
-            .context(invocation::Request)?;
+        let response = match self.client.send(request).await {
+            Ok(response) => response,
+            Err(source @ crate::http::HttpError::CallRequest { .. }) => {
```

Comment:
Just noting for other readers that this appears to be the error we get if any `hyper` error occurs.